src/Patches.php

```diff
@@ -83,6 +83,21 @@ public static function getSubscribedEvents() {
     );
   }
 
+  /**
+   * Turn local patch file uris into absolute paths.
+   * @param string $install_path
+   * @param array $patches
+   */
+  protected function checkUrls($install_path, &$patches) {
+    foreach ($patches as $package_name => $package_patches) {
+      foreach ($package_patches as $description => $url) {
+        if (strpos($url, '.') === 0) {
```

Comment:
I've got it like this, as to not require the patch to start with `./`

```suggestion
        $absolutePath = $install_path . DIRECTORY_SEPARATOR . $url;
        if (file_exists($absolutePath)) {
          $patches[$package_name][$description] = $absolutePath;
        }
```
Either way, both work fine for me!